test/externalTests/zeppelin.sh

```diff
@@ -33,7 +33,7 @@ function zeppelin_test
     OPTIMIZER_LEVEL=1
     CONFIG="truffle-config.js"
 
-    truffle_setup https://github.com/erak/openzeppelin-contracts.git master_060
+    truffle_setup https://github.com/solidity-external-tests/openzeppelin-contracts.git upgrade-0.7.0
```

Comment:
Hm - maybe we should just create a 0.6 branch in the repo then and use it here instead? Also: did you try changing this to the actual upstream repo? I thought zeppelin may actually be compatible with 0.6 already, so if that's possible it would be even better!

Reply:
upgrade-0.7.0 for zeppelin was from the master (although 3 weeks old now) The only additional change is `.gas` and `.value` syntax changes.

Reply:
I can make a 060 version by removing the dot syntax changes.

Reply:
But we can just use https://github.com/OpenZeppelin/openzeppelin-contracts directly here, can't we?
If we can, then it's much better to really check against the upstream repo.

Reply:
Everything except https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/GSN/IRelayHub.sol is up to date with 0.6.0. That file has several unimplemented functions without the keyword virtual.

Reply:
Maybe I can submit a pull reqeust in Zeppelin. We can then change to upstream.

Reply:
You can just pull the branch from @erak's and push it there.

Can we also use a consistent branch naming scheme across all these repos?

Reply:
> Everything except https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/GSN/IRelayHub.sol is up to date with 0.6.0. That file has several unimplemented functions without the keyword virtual.

But it's an interface - we don't require ``virtual`` for interface functions in 0.6, do we?

Reply:
@ekpyron I was wrong; I assumed it because `solidity-upgrade` made those changes. Turns out that `solidity-upgrade` doesn't distinguish between contracts and interfaces when trying to add `virtual` keyword to functions without implementation.

Reply:
Ah, ok - that's not good :-). We should definitely fix solidity-upgrade then... and switch to the upstream repo for the tests after all.

Reply:
It's a quick fix, I think. I can submit a PR for solidity-upgrade on Monday. Currently, this PR uses the upstream repo for Zeppelin!